samples/bluetooth/subrating/README.rst

```diff
@@ -0,0 +1,225 @@
+.. _ble_subrating:
+
+Bluetooth: Connection Subrating
+###############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Bluetooth® Connection Subrating sample showcases the effect of the LE Connection Subrating feature on the duty cycle of a connection.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+You can use any two of the development kits mentioned above and mix different development kits.
+
+Additionally, the sample requires a connection to a computer with a serial terminal for each of the development kits.
+
+Overview
+********
+
+Subrating provides a mechanism for indicating that only a specific subset of connection events are to be actively used by the connected devices, with the radio not being used in other connection events.
```

Comment:
```suggestion
Subrating provides a mechanism for indicating that only a specific subset of connection events can actively used by the connected devices, when the radio not being used in other connection events.
```
Is this still correct? Just guessing here

Reply:
Reworded `are to` as `will be` to keep the wording a bit stricter per spec, but I left `with` as I think the meaning changes slightly otherwise, what do you think?